osf/management/commands/export_user_account.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+import io
+import os
+import json
+import logging
+import requests
+import shutil
+
+from django.core import serializers
+from django.core.management.base import BaseCommand
+
+from addons.wiki.models import NodeWikiPage
+from addons.osfstorage.models import OsfStorageFileNode
+from framework.auth.core import Auth
+from osf.models import (
+    OSFUser,
+    PreprintService,
+    Registration,
+)
+from scripts.utils import Progress
+from website.util import waterbutler_api_url_for
+
+PREPRINT_EXPORT_FIELDS = [
+    'is_published',
+    'date_created',
+    'date_modified',
+    'date_published'
+]
+
+NODE_EXPORT_FIELDS = [
+    'title',
+    'is_fork',
+    'category',
+    'is_public',
+    'description',
+    'forked_date',
+    'date_created',
+    'date_modified'
+]
+
+REGISTRATION_EXPORT_FIELDS = NODE_EXPORT_FIELDS + [
+    'registered_data',
+    'registered_meta'
+]
+
+logging.getLogger('urllib3').setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+
+ERRORS = []
+
+def export_metadata(node, current_dir):
+    """
+    Exports the pretty printed serialization of a given model instance to metadata.json.
+    Only simple fields (non-FK, non-M2M, etc) are serialized.
+
+    """
+    export_fields = NODE_EXPORT_FIELDS
+    if isinstance(node, Registration):
+        export_fields = REGISTRATION_EXPORT_FIELDS
+    elif isinstance(node, PreprintService):
+        export_fields = PREPRINT_EXPORT_FIELDS
+    with open('{}metadata.json'.format(current_dir), 'w') as f:
+        # only write the fields dict, throw away pk and model_name
+        metadata = serializers.serialize('json', [node], fields=export_fields)
+        f.write(json.dumps(metadata, indent=4, sort_keys=True))
```

Comment:
Yeah, I think I just missed that. I also need to remove pk and model_name and just write the fields and values.